src/main/java/shop/service/impl/ProductServiceImpl.java

```diff
@@ -1,4 +1,42 @@
 package shop.service.impl;
 
-public class ProductServiceImpl {
+import shop.dao.ProductDAO;
+import shop.dao.impl.ProductDAOImpl;
+import shop.entity.Product;
+import shop.service.ProductService;
+
+import java.util.*;
+
+public class ProductServiceImpl implements ProductService {
+
+    private ProductDAO productDAO = new ProductDAOImpl();
+
+    @Override
+    public void getProductByName(String name) {
+       Product product = productDAO.findProductByName(name);
+        if (product!=null) System.out.println(product);
+        else System.out.println("No products were found");
+    }
+
+    @Override
+    public void getAllProducts() {
```

Comment:
same with this. return from the method list. And print it in the menu or show a message if the list is empty